src/components/Board.js

```diff
@@ -3,29 +3,33 @@ import './Board.css';
 import Square from './Square';
 import PropTypes from 'prop-types';
 
-
+// This turns the 2D array into a 1D array
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
-
-}
+  const singleArraySquares = [].concat(...squares);
+  return singleArraySquares.map((square) => {
+    return (
+      <Square
+        value={square.value}
+        id={square.id}
+        onClickCallback={onClickCallback}
+        key={square.id}
+      />
+    );
+  });
+};
 
 const Board = ({ squares, onClickCallback }) => {
   const squareList = generateSquareComponents(squares, onClickCallback);
   console.log(squareList);
```

Comment:
When pushing code to be reviewed, we want to make sure we eliminate any print statement or console logs
```suggestion

```